erizo/src/erizo/rtp/RtpRetransmissionHandler.cpp

```diff
@@ -1,14 +1,20 @@
 #include "rtp/RtpRetransmissionHandler.h"
+
+#include <algorithm>
+
 #include "rtp/RtpUtils.h"
 
 namespace erizo {
 
 DEFINE_LOGGER(RtpRetransmissionHandler, "rtp.RtpRetransmissionHandler");
 
-RtpRetransmissionHandler::RtpRetransmissionHandler()
-  : connection_{nullptr},
+RtpRetransmissionHandler::RtpRetransmissionHandler(std::shared_ptr<erizo::Clock> the_clock)
+  : clock_{the_clock},
+    connection_{nullptr},
     audio_{kRetransmissionsBufferSize},
-    video_{kRetransmissionsBufferSize} {}
+    video_{kRetransmissionsBufferSize},
+    bucket_{static_cast<uint64_t>(300000. * kMarginRtxBitrate), kBurstSize, clock_},
```

Comment:
Consider making 300000  another constant. I'm assuming it's 300kbps because of the fast start and it's more or less an _unofficial_ standard.